index.js

```diff
@@ -805,14 +807,16 @@ Cartero.prototype.writeMetaDataFile = function( callback ) {
 	}, {} );
 
 	var metaData = JSON.stringify( {
-		formatVersion : 3,
+		formatVersion : 4,
 		packageMap : packageMap,
 		entryPointMap : entryPointMap,
+		entryPoints : _this.entryPoints,
```

Comment:
Hi @dgbeck, I double check and this should be call `assetsRequiredByEntryPoint` with this structure:
```javascript
assetsRequiredByEntryPoint : {
  'view/blah/blah.es' : {
    script: [ ],
    style: [ ],
    image: [ ]
  }
}
```

I would change this.

Thx!